crates/cheatcodes/src/inspector.rs

```diff
@@ -1175,6 +1177,11 @@ impl Inspector<&mut dyn DatabaseExt> for Cheatcodes {
         if self.gas_metering.paused {
             self.gas_metering.paused_frames.push(interpreter.gas);
         }
+
+        // `expectRevert`: track the max call depth during `expectRevert`
+        if let Some(ref mut expected) = self.expected_revert {
```

Comment:
make @DaniPopes happy hehe
```suggestion
        if let Some(expected) = &mut self.expected_revert {
```